### PR TITLE
update build and .yaml for 0.4

### PR DESCRIPTION
**Pull request by lavalamp**
Also add a service to expose the queue status.

### CODE REVIEW

**Comment by lavalamp** [comment]
@eparis @ixdy

File: submit-queue/pod.yaml [hunk 1]
```diff
@@ -3,9 +3,12 @@ kind: Pod
 metadata:
   name: submit-queue
   namespace: default
+  labels:
+    app: submit-queue
+    version: "0.4"
```
Comment (eparis) [inline submit-queue/pod.yaml]: I'll have to think about how to generate this in the makefile, where we know the Tag

**Comment by eparis** [comment]
#55 called something 0.5.  not sure if brendan actually is running something based on that...

**Comment by lavalamp** [comment]
Huh, that is weird. Well, the 0.4 that's running now is one I built!

**Pull request merged**